xbmc/FileItem.cpp

```diff
@@ -1044,6 +1044,14 @@ bool CFileItem::IsInternetStream(const bool bStrictCheck /* = false */) const
   return URIUtils::IsInternetStream(m_strPath, bStrictCheck);
 }
 
+bool CFileItem::IsStreamedFileSystem() const
+{
+  if (!m_strDynPath.empty())
+    return URIUtils::IsStreamedFilesystem(m_strDynPath);
+
+  return URIUtils::IsStreamedFilesystem(m_strPath);
```

Comment:
CFileItem::GetDynPath imo checks whether m_strDynPath is empty and in case it returns m_strPath. So, this could be simplified to read `return URIUtils::IsStreamedFileSystem(GetDynPath());`

Reply:
Yes, but all the other methods around it use the way I cp-ed so I'd rather keep it all the same....

Reply:
> I'd rather keep it all the same....

No hard feelings, but why? Not saying you should fix all the other methods, though.

Reply:
I like consistency (and less chance of fallout/regression in the future) ;-) But no problem changing it anyway.